src/app/core/data/dso-redirect.service.ts

```diff
@@ -104,6 +108,15 @@ export class DsoRedirectService {
             }
           }
         }
+        // Redirect to login page if the user is not authenticated to see the requested page
+        if (response.hasFailed && (response.statusCode === 401 || response.statusCode === 403)) {
+          // Remove `/` from the namespace if it is empty
+          const namespace = this.appConfig.ui.nameSpace === '/' ? '' : this.appConfig.ui.nameSpace;
```

Comment:
why?

Reply:
Because that URL would look like this: `dev-5.pc//home` if the '/' is not removed.